docs/03-extensions/payzen/how-to/front-commerce.mdx

```diff
@@ -0,0 +1,88 @@
+---
+title: Use as a Front-Commerce payment
+description:
+  "This guides explains how to configure the Payzen extension to use it with
+  Front-Commerce payment methods."
+sidebar_position: 3
+---
+
+<p>{frontMatter.description}</p>
+
+## Front-Commerce configuration
+
+[After installing the Payzen package](/docs/3.x/extensions/payzen/#installation),
+you need to enable the corresponding extension. For that, you need to tweak your
+`front-commerce.config.ts` file like:
+
+```typescript title="front-commerce.config.ts"
+import { defineConfig } from "@front-commerce/core/config";
+import themeChocolatine from "@front-commerce/theme-chocolatine";
+import magento2 from "@front-commerce/magento2"; // or magento1
+// highlight-next-line
+import payzen from "@front-commerce/payzen";
+import storesConfig from "./app/config/stores";
+import cacheConfig from "./app/config/caching";
+
+export default defineConfig({
+  extensions: [
+    magento2({ storesConfig }),
+    themeChocolatine(),
+    // highlight-next-line
+    payzen(), // ⚠️ need to be after themeChocolatine()
+  ],
+  stores: storesConfig,
+  cache: cacheConfig,
+  configuration: {
+    providers: [],
+  },
+});
+```
+
+## Add the required environment variables
+
+In the .env, you need to define the following environment variables:
+
+```shell title=".env"
+FRONT_COMMERCE_PAYZEN_PUBLIC_KEY=MyPayzenPublicKey
+FRONT_COMMERCE_PAYZEN_PRIVATE_KEY=MyPayzenPrivateKey
+FRONT_COMMERCE_PAYZEN_SHA256=MyPayzenSha256
+FRONT_COMMERCE_PAYZEN_PRODUCT="payzen" # use lyra_collect to switch to Lyra Collect's API URLs
+```
+
+## Register your Adyen payment components
+
+:::note
+
+This is a temporary way to setup payment components. We are aware that it is
+tedious. It will definitely change in the future, when Front-Commerce will
+support new extension points for Payments.
+
+:::
+
+1. Override the file that lets you register additional payments forms in
+   Front-Commerce
+
+   ```shell
+   mkdir -p app/theme/modules/Checkout/Payment/AdditionalPaymentInformation/
+   cp -u node_modules/@front-commerce/theme-chocolatine/theme/modules/Checkout/Payment/AdditionalPaymentInformation/getAdditionalDataComponent.js app/theme/modules/Checkout/Payment/AdditionalPaymentInformation/getAdditionalDataComponent.js
+   ```
+
+2. Register Payzen
+
+   ```diff title="app/theme/modules/Checkout/Payment/AdditionalPaymentInformation/getAdditionalDataComponent.js"
+   +import PayzenEmbeddedForm from "theme/modules/Checkout/Payment/AdditionalPaymentInformation/PayzenEmbeddedForm";
+
+   const ComponentMap = {
+    payzen_embedded: PayzenEmbeddedForm
+   };
+
+   const getAdditionalDataComponent = (method) => {
+   +  if (method.code.startsWith("adyen_")) {
+   +    return AdyenCheckout;
+   +  }
+     return ComponentMap[method.code];
+   };
+   ```
+
```

Comment:
I've made the update in the core since the module is M1 only, i belive we should handle this by our side in the chocolatine theme and it don't need to be documented